demo/DemoFull.tsx

```diff
@@ -24,17 +24,18 @@ import { Grid, FormControl, InputLabel, Select, MenuItem, TextField, Typography
 
 
 function Demo() {
-    const examples = {'Bar chart': barChart,
-                      'Heatmap': heatmap,
-                      'Matrix': matrix,
-                      'Comparison of four samples': doubleMarks,
-                      'Annotated chart': ruleMark,
-                      'Comparing two samples': compareTwoSamples,
-                      'Linked views': brush,
-                      'Circular halves': circularHalves,
-                      'Gene annotations': geneAnnotation,
-                      'Ideogram expression': ideogramWithArea,
-                    };
+    const examples = {
+        'Bar chart': barChart,
+        'Heatmap': heatmap,
+        'Matrix': matrix,
+        'Comparison of four samples': doubleMarks,
+        'Annotated chart': ruleMark,
+        'Comparing two samples': compareTwoSamples,
+        'Linked views': brush,
+        'Circular halves': circularHalves,
+        'Gene annotations': geneAnnotation,
+        'Ideogram expression': ideogramWithArea,
+    };
```

Comment:
I probably had to turn off my auto-linting. All formatting changes reflect the `.eslintrc.cjs` option defined in this project.